src/backend/shortcuts/utils.ts

```diff
@@ -68,6 +68,20 @@ async function getIcon(appName: string, gameInfo: GameInfo) {
   let icon = `${iconsFolder}/${appName}.jpg`
 
   if (gameInfo.runner === 'gog') {
+    const icoPath = join(
+      gameInfo.install.install_path!,
+      `goggame-${appName}.ico`
+    )
+    const linuxNativePath = join(
+      gameInfo.install.install_path!,
+      'support',
+      'icon.png'
+    )
+    if (existsSync(icoPath)) {
+      return icoPath
+    } else if (existsSync(linuxNativePath)) {
+      return linuxNativePath
+    }
```

Comment:
While being here I decided to use icons extracted by installer first instead of downloading them. I guess we could copy them over to some Heroic location just to be save they don't disappear